dvc/utils/pager.py

```diff
@@ -12,8 +12,8 @@
 
 
 DEFAULT_PAGER = "less"
-DEFAULT_PAGER_FORMATTED = "{} --chop-long-lines --clear-screen".format(
-    DEFAULT_PAGER
+DEFAULT_PAGER_FORMATTED = (
+    f"{DEFAULT_PAGER} --chop-long-lines --clear-screen --RAW-CONTROL-CHARS"
```

Comment:
note that we want the capital `-R` option and not lowercase `-r` to only enable output for color escape sequences (lowercase `-r` enables raw output for all terminal escape sequences)